R/editing.R

```diff
@@ -367,6 +367,15 @@ publish_update <- function(mn,
     # Get the metadata doc
     message("Getting metadata from the MN.")
     eml <- EML::read_eml(rawToChar(dataone::getObject(mn, metadata_pid)), asText = TRUE)
```

Comment:
Need to close this if statement correct?

Reply:
looking into it.